From 50d16cd06e17fbab0669f394b7eadbba4a612a70 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Mon, 25 Nov 2019 11:55:25 -0600
Subject: [PATCH 30/34] ARM64: dts: logicpd-imx8mm-kit: remove unwanted
 synaptics muxing

The baseboard shows an entry for pinctrl_i2c2_synaptics_dsx_io
which does not exist on the Logic PD platform.

This patch removes the unnecessary mux entry.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
index 5aa64f0f2d49..84dbdbdcb662 100644
--- a/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
@@ -110,12 +110,6 @@
 			>;
 		};
 
-		pinctrl_i2c2_synaptics_dsx_io: synaptics_dsx_iogrp {
-			fsl,pins = <
-				MX8MM_IOMUXC_GPIO1_IO09_GPIO1_IO9		0x19	/* Touch int */
-			>;
-		};
-
 		pinctrl_i2c2: i2c2grp {
 			fsl,pins = <
 				MX8MM_IOMUXC_I2C2_SCL_I2C2_SCL			0x400001c3
-- 
2.17.1

